docs/variant_annotation.md

```diff
@@ -121,19 +133,16 @@ annotations; use this field to override the field name.
 * [`--vep_num_fork`](https://github.com/googlegenomics/gcp-variant-transforms/blob/c4659bba2cf577d64f15db5cd9f477d9ea2b51b0/gcp_variant_transforms/options/variant_transform_options.py#L208)
 number of forks when running VEP, see [`--fork` option of VEP](ensembl.org/info/docs/tools/vep/script/vep_options.html#opt_fork).
 
+
 For other parameters, like how many VMs to use or where the VMs should be
 located, the same parameters for
 [Dataflow pipeline execution](https://cloud.google.com/dataflow/pipelines/specifying-exec-params)
-are reused, e.g., `--num_workers`.
+are reused, e.g., `--num_workers`. It is recommended to provide at least 100 
+workers for large inputs. Further increasing the number of workers can reduce 
```

Comment:
What is large inputs? If we cannot define it, this comment is not very helpful.

Alternatively we can give a rule of thumb to users, like 1 worker per 50k (or '--number_of_variants_per_shard') variants, can we?